src/box/memtx_engine.cc

```diff
@@ -1358,6 +1358,31 @@ memtx_engine_complete_join(struct engine *engine, void *arg)
 	free(ctx);
 }
 
+#if defined(ENABLE_CHECKPOINT_JOIN)
+#include "memtx_checkpoint_join.c"
+#else /* !defined(ENABLE_CHECKPOINT_JOIN) */
+
+/**
+ * Feed the .snap file with signature, taken from vclock, starting
+ * from the given lsn to the stream.
+ */
+static int
+memtx_engine_checkpoint_join(struct engine *engine, const struct vclock *vclock,
+			     int64_t *lsn, int64_t start_lsn,
+			     struct xstream *stream)
+{
+	(void)engine;
+	(void)vclock;
+	(void)lsn;
+	(void)start_lsn;
+	(void)stream;
+	diag_set(ClientError, ER_UNSUPPORTED, "Tarantool CE",
+		 "checkpoint snapshot fetching");
```

Comment:
Nit: This function isn't going to be called in CE so you could use the generic stub:

```C
#define memtx_engine_checkpoint_join generic_engine_checkpoint_join
```

Reply:
Done